File: pgm/ABC_Visual_Basic/ABC_136_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B, C As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1) : C = arr(2)
        Console.WriteLine(If(A < B + C, B + C - A, 0))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_136_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int A, B, C;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        C = int.Parse(arr[2]);
        Console.WriteLine(A < B + C ? B + C - A : 0);
        return;
    }
}